src/python/bot/fuzzers/libFuzzer/constants.py

```diff
@@ -66,3 +66,41 @@
 
 # libFuzzer's exit code if a bug was found in the target code.
 TARGET_ERROR_EXITCODE = 77
+
+FUCHSIA_QEMU_COMMAND_TEMPLATE = ['{qemu}',
```

Comment:
I don't think these command templates should go in constants.py, I think they should go in libfuzzer.py (part of the FuchsiaQemuLibFuzzerRunner class IMO). I personally don't like the idea of a file called constants.py since it is confusing (it seems like a bad idea to put all of one's constants in one file, all functions in another etc.).
However, this file is only supposed to have constants that are meaningful to libFuzzer. These commands are not that IMO.

Reply:
moved!